gems/smithy/spec/interfaces/welds/auth_schemes_spec.rb

```diff
@@ -48,7 +48,7 @@ def remove_auth_schemes
       end
 
       it 'removes auth schemes from the client' do
-        expect(client.config.auth_schemes).not_to include('smithy.api#httpBearerAuth')
+        expect(client.config.auth_schemes).to include('smithy.api#httpBearerAuth')
```

Comment:
I don't think this test file makes too much sense now, since we no longer add auth schemes via code generation. This test would be wrong otherwise?